specification/specs/cluster/cluster_delete_voting_config_exclusions/ClusterDeleteVotingConfigExclusionsResponse.ts

```diff
@@ -17,6 +17,6 @@
  * under the License.
  */
 
-class ClusterDeleteVotingConfigExclusionsResponse extends ResponseBase {
-  stub: integer
-}
+class ClusterDeleteVotingConfigExclusionsResponse
+  extends ResponseBase
+  implements EmptyResponseBase {}
```

Comment:
I'm not sure `EmptyResponseBase` should be used.
Usually, `EmptyResponseBase` represents what any `HEAD` API would return. What is `cluster.delete_voting_config_exclusions` returning?

Reply:
`cluster.delete_voting_config_exclusions` response is empty, solely the HTTP status code is the indicator of the operations status.